src/app/components/table-row-option.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component, Input, Output, EventEmitter } from "@angular/core";
+
+@Component({
+  selector: "app-table-row-option",
+  templateUrl: "table-row-option.component.html",
+})
+export class TableRowOptionComponent {
```

Comment:
What do you think of following the Angular Material component design with a directive here? Would allow for using links should they be needed.